recipes/r-changeforest/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set name = "r-changeforest" %}
+{% set version = "0.1.1" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mlondschien/changeforest/archive/refs/tags/{{ version }}.tar.gz
+  # TODO: Both
+  # https://github.com/mlondschien/changeforest/releases/download/{{ version }}/changeforest-{{ version }}.tar.gz
+  # and
+  # https://github.com/mlondschien/changeforest/releases/download/{{ version }}/{{ version }}.tar.gz
+  # don't work.
+  sha256: 036ab5c1ec8dcfadfe9eee5fddd4c8fbd3b2c7a622fa570f2bbcba776404de6e
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Taken from https://github.com/conda-forge/r-gifski-feedstock
+requirements:
+  build:
+    - rust
```

To support cross-compiling.
```suggestion
    - {{ compiler('rust') }}
```